common/build.gradle

```diff
@@ -68,3 +64,21 @@ configurations.all {
     resolutionStrategy.force "joda-time:joda-time:2.10.12"
     resolutionStrategy.force "org.slf4j:slf4j-api:1.7.36"
 }
+
+spotless {
+    java {
+        target fileTree('.') {
+            include '**/*.java'
+            exclude '**/build/**', '**/build-*/**'
+        }
+        importOrder()
+//        licenseHeader("/*\n" +
```

Comment:
if we aren't going to apply this, we should either have a comment or remove it.

Reply:
Good idea, just added a comment that references the blocking issue